### PR TITLE
Create an "Expert Subject Set" with Workflow

### LINKED ISSUE #293: Subject Queue Behaviour/Expert Classifications
Let's figure this out.

### PR DESCRIPTION
**Pull request by edpaget**
Closes #293

### CODE REVIEW

File: spec/controllers/api/v1/workflows_controller_spec.rb [hunk 1]
```diff
@@ -130,6 +130,27 @@
...
 
+    context "creates an expert subject set" do
+      let(:instance) { SubjectSet.find(json_response["workflows"][0]["links"]["expert_subject_set"]) }
+      
+      before(:each) do
+        default_request scopes: scopes, user_id: authorized_user.id
+        post :create, create_params
+      end
+      
+      it 'shold have a link to the created set' do
```
Comment (camallen) [inline spec/controllers/api/v1/workflows_controller_spec.rb]: should spelling

File: spec/controllers/api/v1/workflows_controller_spec.rb [hunk 2]
```diff
@@ -130,6 +130,27 @@
       end
     end
 
+    context "creates an expert subject set" do
+      let(:instance) { SubjectSet.find(json_response["workflows"][0]["links"]["expert_subject_set"]) }
```
Comment (camallen) [inline spec/controllers/api/v1/workflows_controller_spec.rb]: Extract common response object to it's own let, perhaps:
```ruby 
let(:respone_ess_id) { json_response["workflows"][0]["links"]["expert_subject_set"] }
let(:instance) { SubjectSet.find(respone_ess_id) }
```

File: spec/controllers/api/v1/workflows_controller_spec.rb [hunk 3]
```diff
@@ -130,6 +130,27 @@
...
+      
+      before(:each) do
+        default_request scopes: scopes, user_id: authorized_user.id
+        post :create, create_params
+      end
+      
+      it 'shold have a link to the created set' do
+        expect(json_response["workflows"][0]["links"]["expert_subject_set"]).to_not be_nil
```
Comment (camallen) [inline spec/controllers/api/v1/workflows_controller_spec.rb]: use the common let `respone_ess_id`

**Comment by camallen** [comment]
Looks good

**Comment by edpaget** [comment]
Changes made.

**Pull request merged**